.copyright

```diff
@@ -0,0 +1,3 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
```

Comment:
Looks like the tests are failing because of the copyright year. Some of the samples are created years before.

Reply:
@RedBrogdon & @domesticmouse would suggest something.